pandas/core/indexes/base.py

```diff
@@ -5508,7 +5508,10 @@ def drop(self, labels, errors: str_t = "raise"):
         """
         arr_dtype = "object" if self.dtype == "object" else None
         labels = com.index_labels_to_array(labels, dtype=arr_dtype)
-        indexer = self.get_indexer(labels)
+        if self.is_unique:
+            indexer = self.get_indexer(labels)
+        else:
+            indexer, _ = self.get_indexer_non_unique(labels)
```

Comment:
get_indexer_for?

Reply:
Did not know this function, thanks very much.